sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsMeta.scala

```diff
@@ -494,6 +519,21 @@ abstract class SparkPlanMeta[INPUT <: SparkPlan](plan: INPUT,
   override val childParts: Seq[PartMeta[_]] = Seq.empty
   override val childDataWriteCmds: Seq[DataWritingCommandMeta[_]] = Seq.empty
 
+  var cpuCost: Double = 0
+  var gpuCost: Double = 0
```

Comment:
I'm not sure there's a point to having getters and setters if these variables aren't private.  Personally I'm OK if we want to go the simple route and just expose them for now, we can always place them behind accessors later since this isn't part of a public API.